.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ parameters:
 executors:
   integration-test-validator:
     docker:
-      - image: cimg/openjdk:17.0
+      - image: maven:3.9.5-eclipse-temurin-21
```

Comment:
Seems like an odd choice of image... why do you need Maven installed on it? The Kotlin template uses one of our executors...

Reply:
thanks. I will update that.. was copied from a different project.

Reply:
Updated to use cimg/openjdk:21.0.0